.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,6 +39,7 @@ Of course, you can opt not to do this and be listed as an "Anonymous contributor
 - [ ] I have made corresponding changes to the documentation if needed.
 - [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
 - [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
+- [ ] I have linked any issues that are relevant to this PR. <!-- Put Closes: #<issue number> in the description section of this PR. -->
```

Comment:
Super excellent suggestion @ethan-master-coding! This will definitely help out the whole workflow 😄 

What do we think about the following suggestion?

```
- [ ] I have linked any issues that are relevant to this PR (Write "Closes #<issue number> in the "Description" below).
```

This way, this direction is both visible onscreen and can be caught by screenreaders?